source/sass/mozfest.scss

```diff
@@ -11,6 +11,8 @@ body.mozfest {
   .tito-widget-button {
     background: unset;
     color: unset;
+    font-weight: unset;
+    padding: unset;
```

Comment:
The `tito-button` was looking different than other buttons because of some undesired overrides.